integration-tests/call-builder-return-value/lib.rs

```diff
@@ -0,0 +1,293 @@
+//! # Integration Tests for `LangError`
```

Comment:
Why can't his example be incorporated into `call-builder-delegate` example?

Reply:
Different storage layouts are used in each